### ISSUE TITLE
Blinds needed for at least South facing windows.

### ISSUE THREAD

**Issue by magman2112** [DoESLiverpool/somebody-should#701]
Due to the level of sunlight expected through the South Facing windows, it is anticipated that these windows as a minimum will need blinds. Anita mentioned last week that another resident of Tapestry, MD, has already spoken to a supplier. The supplier has suggested a possible significant discount for a larger order. Someone needs to discuss this with the MD folks.

* Confirm that blinds are required on South Facing windows
* Measure windows 
* Decide whether any windows on North side also need blinds (Meeting room perhaps).

**Comment by amcewen** [comment]
This isn't needed before we move in, so dropping the `DoES Move: The Tapestry` label

**Comment by jackie1050** [comment]
Would stencilling the windows with Processing and something semi-permanent like white chalk spray be enough to reduce glare, at least as a short term measure?

**Comment by johnmckerrell** [comment]
Have called Solarshade Blinds who are the people on London Road to get a quote, hopefully they'll be around tomorrow. We appear to have used Factory Blinds Limited previously but they've now gone out of business (The Blinds Factory appears to be a different entity).

**Comment by johnmckerrell** [comment]
Solarshade have been in to measure up and will send through a quote for the south facing windows.

**Comment by johnmckerrell** [comment]
Actually the quote was for blackout blinds in there. I forgot we’d want different. We’ll need to find out some options. 

--

**Comment by johnmckerrell** [comment]
Called today but was asked to call back in the morning, I won't be around tomorrow so will call first thing on Thursday.

**Comment by amcewen** [comment]
Blackout blinds should be being fitted to the events space next Wednesday.

**Comment by amcewen** [comment]
The blackout blinds have been fitted.

**Comment by johnmckerrell** [comment]
We've had Solarshade in again to discuss Room 29. It's (still) £750 for black out blinds, apparently "dim out" would be cheaper but I haven't established by how much, we were considering whether that will be dim enough or if having bright shining white sheets on the window would still be too much. I have a colour sample book if we'd like to try some colours rather than sticking with white.

I also got a quote for window film, actually a lot more expensive at £1800+VAT, that would have given a dark tint to the windows while leaving them see through.

**Comment by johnmckerrell** [comment]
I was recommended against vertical blinds as they get caught up on window handles and things and get in the way. Venetian blinds would get very dusty.

**Comment by zarino** [comment]
@johnmckerrell do you know what dimensions that they’re quoting for?

**Comment by johnmckerrell** [comment]
For window film or blinds? Not off the top of my head but it’s 6 blinds to span the room, same as in the events room.

**Comment by amcewen** [comment]
I'm not sure where the grey stuff is, but they're now experimenting with the brown.

**Comment by johnmckerrell** [comment]
The panes do seem to vary in size:

1 | 2 | 3
-- | -- | --
655mm x 845mm | 680mm x 845mm | 660mm x 845mm
655mm x 840mm | 585mm x 775mm | 660mm x 870mm
655mm x 840mm | 680mm x 840mm | 660mm x 840mm

(I've measured one set of 3 x 3 windows and assuming the others are the same but it's quite possible that's not the case given how much the width varies on that 3 x 3)

**Note that these are still approximate measurements and shouldn't be used for cutting.** An example of film available online would be:

[ebay link](https://www.ebay.co.uk/itm/NEUTRAL-UV-60-ULTRAVIOLET-99-5-PROTECTION-PRO-WINDOW-TINTING-TINT-FILM/261345844127?hash=item3cd96d1b9f:m:mCm0uZdTxDCxJFytAZ4qtXA)

845 (length) * 36 (panels) = 30,420mm / 2 (width is enough to fit two) = 15,210mm needed = 15.2m

So £235 - £309 to do the lot if I've done my maths right there, perhaps someone could confirm. That actually seems quite doable if we feel we can do a good enough job of applying the stuff. Would be £29-39 for a shorter length to try doing one strip and see if we feel we could do the whole lot.

**Comment by magman2112** [comment]
In the dim and distant past, I actually watched someone fit this type of film to a window and it is definitely do-able. 

 

The windows have to be really clean though, and you get the film to be oversized for the window pane and trim it in place with a sharp knife. You use a spray bottle of water with a little detergent to apply the film, then squeegee the water out to the edge to prevent bubbles in the finish. Working up a ladder for the higher windows will be a bit of a challenge as well methinks.

 

We would likely need to get some good squeegee tools (a wooden handle with a rubber insert for the blade) as well as a stack of sharp knife blades. More items to look out for on eBay methinks. 

 

How about trying the few windows on the Eastern side of Room 29 as a test, as these windows likely need to be treated anyway?

 

Regards

 

Mike

**Comment by ajlennon** [comment]
If somebody wants to prod me with dates I could try and help out...

**Comment by ajlennon** [comment]
(The numeric kind as opposed to the delicious edible kind, although those would probably not go amiss either)

**Comment by johnmckerrell** [comment]
If we think it’s a goer then we probably want to shop around some more and see which we think would work best. 

--

**Comment by ajlennon** [comment]
Probably medjoul I am guessing

**Comment by johnmckerrell** [comment]
@magman2112 I don't think we'll need to do the Eastern windows at all, or at least certainly not until next summer. They're also actually fairly high now I look at them. I think we could just pick some of the lowest ones and give it a go. With the link I posted we'd probably get a 2m roll which would be enough for 3 panes, see what we thought of that and if we liked it then go for the 15m or 20m. We still need to shop around to make sure we have the right film and that the right film is affordable.

**Comment by zarino** [comment]
I don‘t have any expertise to offer with window film, but I’m happy to help when you’re applying it.

**Comment by johnmckerrell** [comment]
Jackie had some window film which I tried but it really didn't do the trick for the direct sunlight. I think we'll need to stick with blinds.

@mdunschen has mentioned blinds that come up from the bottom as well as down from the top which could be handy given that people near the windows will have different needs from those on the other side of the room but he's not been able to find them in the UK. Anyone aware of anything? (Not checked with Solarshade yet)

**Comment by johnmckerrell** [comment]
It's been suggested that we might just order the blinds online. Looking at these with approximate measurements suggests we could actually make a big saving, around £480 inc VAT for the ones we'd need (assuming they're the same as the ones we've been quoted).

https://www.blinds-2go.co.uk/roller-blinds/5853/sevilla-tranquility-brilliant-white.html

I also found this one which can be positioned anywhere in the window but unfortunately wouldn't do our drop as our windows are so long.

https://www.blinds-2go.co.uk/top-down-bottom-up-duo/20224/duoshade-dove.html?gclid=Cj0KCQjwjbveBRDVARIsAKxH7vm_Nta7eQNwWZb_3XiHjmjqdeGFbURLTf17IerUKh-lQ0vyoyVCcnoaAk1fEALw_wcB

**Comment by johnmckerrell** [comment]
https://www.blinds-2go.co.uk/roller-blinds/734/white.html

Left window 2820mm high by 4550mm wide, split 1515, 1520, 1515 = 4,550
Right window 2820mm high by 4538mm wide, split 1509, 1520, 1509 = 4,538

I'd basically be happy ordering from these measurements, I was wondering if we could standardise a bit but the outer ones differ by a whole half a cm so I'm not sure that would be possible, the inner ones (1520) can be the same. I'll see what the event room is later.

**Comment by johnmckerrell** [comment]
Forgot to say in the previous comment that I'm now happier that non-blackout blinds will be ok. I'll need to confirm which side the controls need to drop (I was originally thinking to match the events room but optimising for how we're placing desks might actually be better). With these measurements though we're actually looking at only £276.90 all in.

**Comment by johnmckerrell** [comment]
Event room was a bit of a mix of measurements, I only measured the left 4 blinds but they were something along the lines of A, B, A, C where A < C < B (I forget, B & C were quite similar).

Controls were also Left, Right, Right, Left, Right, Right in the events room.

Looking at our desk placement I think we'd be better off with Left, Right, Right, Left*, Left, Right. The starred one might be better on the right but would mean they're together. It just depends how much you'd be pulling the cord to the side if you were leaning over and if that would damage the blind by rubbing against it.

**Comment by zarino** [comment]
@johnmckerrell – @jackie1050 and I just measured the widths of the Room 29 windows.

<table>
<tbody>
<tr>
<th colspan="6">Room 29: Left bank</th>
<th colspan="6">Room 29: Right bank</th>
</tr>
<tr>
<td colspan="2" align="center">Pair 1</td>
<td colspan="2" align="center">Pair 2</td>
<td colspan="2" align="center">Pair 3</td>
<td colspan="2" align="center">Pair 4</td>
<td colspan="2" align="center">Pair 5</td>
<td colspan="2" align="center">Pair 6</td>
</tr>
<tr>
<td colspan="2" align="center">1510 mm</td>
<td colspan="2" align="center">1525 mm</td>
<td colspan="2" align="center">1520 mm</td>
<td colspan="2" align="center">1510 mm</td>
<td colspan="2" align="center">1520 mm</td>
<td colspan="2" align="center">1510 mm</td>
</tr>
<tr>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
</tr>
</tbody>
</table>

(Measurements taken up to walls on the outer edges of each bank, and to the centre of the white plastic "pillars" on inner edges.)

So, our measurements for Room 29 came out within 5 mm of your original measurements. Close enough.

Just for kicks, I also measured the widths of the Event room windows *and* blinds

<table>
<tr>
<th colspan="6">Event room: Left bank</th>
<th colspan="6">Event room: Right bank</th>
</tr>
<tr>
<td colspan="2" align="center">Pair 1</td>
<td colspan="2" align="center">Pair 2</td>
<td colspan="2" align="center">Pair 3</td>
<td colspan="2" align="center">Pair 4</td>
<td colspan="2" align="center">Pair 5</td>
<td colspan="2" align="center">Pair 6</td>
</tr>
<tr>
<td colspan="2" align="center">1510 mm</td>
<td colspan="2" align="center">1520 mm</td>
<td colspan="2" align="center">1510 mm</td>
<td colspan="2" align="center">1510 mm</td>
<td colspan="2" align="center">1525 mm</td>
<td colspan="2" align="center">1515 mm</td>
</tr>
<tr>
<td colspan="2" align="center">Blind:<br>1465 mm</td>
<td colspan="2" align="center">Blind:<br>1490 mm</td>
<td colspan="2" align="center">Blind:<br>1465 mm</td>
<td colspan="2" align="center">Blind:<br>1480 mm</td>
<td colspan="2" align="center">Blind:<br>1480 mm</td>
<td colspan="2" align="center">Blind:<br>1480 mm</td>
</tr>
<tr>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
<td align="center">🔲</td>
</tr>
</table>

So, clearly, despite the differing physical widths of the "pairs" in the Right bank of the Events room (1510/1525/1515) Solarshade decided to keep it simple and cut three identical 1480 mm blinds. It looks fine – as long as the blinds themselves extend beyond the edges of the glass panes, and onto the plastic frames, even a little bit, it all works.

For orders based on "recess" measurements, Blinds2Go say they’ll "make a small deduction of 3mm from the width you have entered" which, I think, then gives them the total (outer) width between the outer sides of each bracket. It sounds like they _then_ aim to cut the blinds 35mm narrower than that total bracket-to-bracket width.

This looks fairly similar to what Solarshade have done on the Right bank of the Events room – If you take blinds 1480 mm wide, then add about 38 mm to each of them, you end up with three 1520 mm bracket-to-bracket units, and that’s roughly the same as the 1510+1525+1515 space that needed covering.

Long story short, I think we shouldn’t get too hung up about exact measurements, and should instead order 6 identical blinds that fit _roughly_ into the space, but not necessarily _precisely_ aligned with the edges of the "pairs" of windows.

Perhaps just order 6 identical blinds, with a "recess width" of 1510 mm? That’ll result in blinds around 1470–1480 mm wide, which, even at 1470 mm, _still_ overlaps the edges of the window frames by about 10 mm on each side, meaning light shouldn’t bleed around the edges too much.

**Comment by johnmckerrell** [comment]
Ordered! We have 3 left and 3 right control all 1510 wide. The ones in the event room are positioned with the blind against the window but it might be better to have it dropping away from the window as they won't catch on the window handle so much, and as we're not getting blackout blinds the gaps won't matter so much. If we do this we'll just need to swap them around (given we've got 3 of each).

**Comment by johnmckerrell** [comment]
These have now arrived and will need fitting. I'm not sure when I'd get around to it, I'm happy for anyone to do it that's confident they can do it properly. The ones in the event room are screwed into the window frame, I'm not sure if there's any special considerations for doing that.

**Comment by ajlennon** [comment]
This is one of the many jobs I would be utterly incompetent at, but should be about to give an assist Friday afternoon if that works

**Comment by johnmckerrell** [comment]
Half-way into install, key learnings are:

1. When ordering from blinds2go you need to call them once the order has been made on the website so that they will give you cords that are longer than a metre. The ones we've got are much too short but they're sending longer ones.
2. You need to fit the centre blind first. The one we've got is almost too narrow, I've had to screw in a little closer to the glass than I wanted. Once the centre one is on you'd then do the other two.

Apart from not being able to raise or lower them without a ladder they're looking pretty good. Not tested in direct sunlight yet either though.

**Comment by johnmckerrell** [comment]
These are now all fitted, two screws either side and the long pull cords.
